tests/bson/bson-toPHP-008.phpt

```diff
@@ -0,0 +1,67 @@
+--TEST--
+MongoDB\BSON\toPHP(): Setting fieldPath typemaps for compound types with string keys
+--FILE--
+<?php
+
+require_once __DIR__ . "/../utils/basic.inc";
+
+class MyArrayObject extends ArrayObject implements MongoDB\BSON\Unserializable
+{
+    function bsonUnserialize(array $data)
+    {
+        parent::__construct($data);
+    }
+}
+
+$bson = \MongoDB\BSON\fromPHP( [
+    '_id' => 1,
+    'array' => [1, 2, 3],
+    'object' => ['string' => 'keys', 'for' => 'ever']
+] );
+
+function fetch($bson, $typeMap = []) {
+    return [ \MongoDB\BSON\toPHP($bson, $typeMap) ];
```

Comment:
Is there any reason for this to return an array if all of the tests have to access the first element before asserting?

Reply:
I had left it so that the assertions were the same as the tests where these were copied from. I'll change them.